src/pykeen/ablation/ablation.py

```diff
@@ -315,8 +327,15 @@ def prepare_ablation_from_config(
     )
 
 
+def path_to_str(x: object) -> str:
+    """Convert path to string and error on everything which is not a path."""
+    if isinstance(x, pathlib.Path):
+        return x.as_posix()
+    raise TypeError(x)
+
+
 def prepare_ablation(  # noqa:C901
-    datasets: Union[str, List[str]],
+    datasets: str | List[str | SplitToPathDict],
```

Comment:
would this also allow for a single SplitToPathDict?

Reply:
The code looks like it does support it; I updated the type hints of `prepare_ablation` and `ablation_pipeline` accordingly (a0281b24)